tests/insert_and_evict_test.cc.cu

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <stdio.h>
+#include "merlin/types.cuh"
+#include "merlin_hashtable.cuh"
+#include "merlin_localfile.hpp"
+#include "test_util.cuh"
+
+constexpr uint64_t DIM = 4;
+using K = int64_t;
+using M = uint64_t;
+using V = float;
+using Table = nv::merlin::HashTable<K, V, M>;
+using TableOptions = nv::merlin::HashTableOptions;
+
+void test_insert_and_evict() {
+  TableOptions opt;
+  size_t bucket_num = 1;
+  size_t bucket_len = 128;
+  size_t keynum = bucket_num * bucket_len;
+  opt.max_capacity = keynum;
+  opt.init_capacity = keynum;
+  opt.max_hbm_for_vectors = keynum * (sizeof(K) + DIM * sizeof(V) + sizeof(M));
```

Comment:
No need to calculate the K and M, because this configurable item is only for vectors.